components/content-service/pkg/storage/gcloud_test.go

```diff
@@ -66,230 +53,3 @@ func TestObjectAccessToNonExistentObj(t *testing.T) {
 		t.Errorf("gcloud storage reported object found despite it being non-existent")
 	}
 }
-
-var runWithDocker = flag.Bool("with-docker", false, "run fake-gcs-server in docker")
-
-func TestObjectUpload(t *testing.T) {
```

Comment:
These tests are not needed anymore?

Reply:
Good question. We are removing the upload code (Go) and using `gsutil` binary instead.